examples/lm_task.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+# test the correct startup of mixed OpenMP / MPI / CPU / GPU tasks
```

Comment:
Should this be in examples if it is a test?

Reply:
I changed the text :-)